webpack.config.babel.js

```diff
@@ -41,12 +48,13 @@ const webpack = {
   devServer: {
     contentBase: path.resolve(__dirname, 'public/'),
     historyApiFallback: true,
-    port: 3000,
+    port: 4000,
```

Comment:
Why have you changed the port number?

Reply:
Previously backend and frontend are running on localhost port 3000. It is not possible to run both ports at the same time. Best way to do so is to name port differently.